client/app/src/Controller/Admin/ReportSubmissionController.php

```diff
@@ -80,9 +81,12 @@ public function indexAction(Request $request, ParameterStoreService $parameterSt
 
         $currentFilters = self::getFiltersFromRequest($request);
         $ret = $this->restClient->get('/report-submission?'.http_build_query($currentFilters), 'array');
-
         $records = $this->restClient->arrayToEntities(EntityDir\Report\ReportSubmission::class.'[]', $ret['records']);
 
+        foreach ($records as $reportSubmission) {
+            $report = $reportSubmission->getReport();
```

Comment:
Not sure if this change is needed, can't see any use of the $report variable here

Reply:
Ha, yes thank you. I had that in from before for some debugging and removed the printy bit but not that! Removed that block now. Cheers